### ISSUE TITLE
Migration to TF 2.0

### ISSUE THREAD

**Issue by nikita0511** [bgshih/aster#109]
Hi,

Is there any plans to migrate the repo to TF v2.0? Currently it runs for versions < 1.14 but fail for higher versions. 

Thanks,
Nikita

**Comment by old-school-kid** [comment]
Yeah it is now supported by TF v2.0. You can download the tf2 weight file and follow the instruction from [here](https://github.com/bgshih/aster/tree/master/tf2_utils)

**Comment by skbose** [comment]
Hello @old-school-kid, is the upgraded training pipeline also available?

Thanks,
Sourav
